tox.ini

```diff
@@ -4,8 +4,7 @@ envlist =
     flake8
     mypy
 
-    # Twisted 15.5 is the first version to support Python 3 (3.3+)
-    coverage-py{27,34,35,36,py2,py3}-tw{166,171,current,trunk}
+    coverage-py{27,34,35,36,37,py2,py3}-tw{166,171,current,trunk}
 
```

Comment:
Comment moved to setup.py, where it is relevant.
py37 added to default env list.